change-notes/1.19/analysis-python.md

```diff
@@ -58,6 +58,7 @@ A new predicate `Stmt.getAnEntryNode()` has been added to make it easier to writ
 |-----------------------------|-----------|--------------------------------------------------------------------|
 | Flask app is run in debug mode (`py/flask-debug`) | security, external/cwe/cwe-215, external/cwe/cwe-489 | Finds instances where a Flask application is run in debug mode. Enabled on LGTM by default. |
 | Information exposure through an exception (`py/stack-trace-exposure`) | security, external/cwe/cwe-209, external/cwe/cwe-497 | Finds instances where information about an exception may be leaked to an external user. Enabled on LGTM by default. |
+| Jinja2 templating with autoescape=False (`py/jinja2/autoescape-false`) | security, external/cwe/cwe-079 | Finds instantiations of jinja2.Environment with `autoescape=False` which may allow XSS attacks. Not enabled on LGTM by default. |
```

Comment:
Ideally all of these queries would include either:

* Results are shown on LGTM by default.
* Results are hidden on LGTM by default.

As on the pages for 1.18, for example: https://help.semmle.com/wiki/display/SD/Python+analysis+1.18.

Talking about queries being enabled is misleading since they run all the time,it's just a question of whether or not the results are displayed by default or not.